app/mailers/candidate_mailer.rb

```diff
@@ -43,57 +43,45 @@ def new_referee_request(reference, reason:)
     )
   end
 
-  def application_rejected_all_rejected(application_choice)
+  def all_applications_rejected(application_choice)
     @course = application_choice.course_option.course
-    @application_choice = application_choice
+    @application_choice = RejectedApplicationChoice.new(application_choice)
     @candidate_magic_link = candidate_magic_link(@application_choice.application_form.candidate)
+    @multiple_applications = application_choice.self_and_siblings.count > 1
 
-    email_for_candidate(
-      application_choice.application_form,
-      subject: I18n.t!(
-        application_choice.rejected_by_default ? 'candidate_mailer.application_rejected_by_default.subject' : 'candidate_mailer.application_rejected.all_rejected.subject',
-      ),
-      template_name: :application_rejected_all_rejected,
-    )
+    email_for_candidate(application_choice.application_form)
   end
 
-  def application_rejected_awaiting_decisions(application_choice)
-    @decisions = application_choice.self_and_siblings.select(&:awaiting_provider_decision?)
-    @application_choice = application_choice
-
-    # We cannot use `through:` associations with FactoryBot's `build_stubbed`. Using
-    # the association directly instead allows us to use `build_stubbed` in tests
-    # and mailer previews.
+  def application_rejected__one_offer_one_awaiting_decision(application_choice)
     @course = application_choice.course_option.course
+    @application_choice = RejectedApplicationChoice.new(application_choice)
+    @offer = application_choice.self_and_siblings.find(&:offer?)
+    @awaiting_decision = application_choice.self_and_siblings.find(&:awaiting_provider_decision?)
+    @awaiting_decision_by = @awaiting_decision.decline_by_default_at.to_s(:govuk_date)
+    @candidate_magic_link = candidate_magic_link(@application_choice.application_form.candidate)
 
-    email_for_candidate(
-      application_choice.application_form,
-      subject: I18n.t!(
-        application_choice.rejected_by_default ? 'candidate_mailer.application_rejected_by_default.subject' : 'candidate_mailer.application_rejected.awaiting_decisions.subject',
-        provider_name: @course.provider.name,
-        course_name: @course.name_and_code,
-      ),
-    )
+    email_for_candidate(application_choice.application_form)
   end
 
-  def application_rejected_offers_made(application_choice)
-    @offers = application_choice.self_and_siblings.select(&:offer?)
-    @decline_by_default_at = @offers.first.decline_by_default_at.to_s(:govuk_date)
-    @dbd_days = @offers.first.decline_by_default_days
-    @application_choice = application_choice
+  def application_rejected__awaiting_decision_only(application_choice)
```

Comment:
Is there a reason for the double underscore? It's more noticeable when viewing the list of previewable email methods as we don't appear to use this elsewhere. Not a blocker as it's coding style but I don't understand how this method differs from other rejection email methods without a double underscore.

Reply:
The main reason I put this in place was so I could distinguish between the old and new emails, and it also made it a bit easier to understand the email context, but happy to update the names now that the old emails are out.